pkgs/development/python-modules/pypager/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  prompt_toolkit,
+  pygments,
+}:
+
+let
+  pname = "pypager";
+in
+buildPythonPackage {
+  inherit pname;
```

Comment:
Please don't re-use `pname.

`<something> = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21